doc/en/api/1.0.0/gwclayers.yaml

```diff
@@ -0,0 +1,236 @@
+          description: The name of the layer to retrieve.
+          type: string
+        - name: quietOnNotFound
+          in: query
+          required: false
+          description: The quietOnNotFound parameter avoids logging an exception when the layer  is not present. Note that 404 status code will still be returned.
+          type: boolean
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/Layer"
+          examples:
+            application/xml: |
+              <GeoServerLayer>
+                <id>LayerInfoImpl--3a33e7d0:1400d3d823c:-7fdf</id>
+                <enabled>true</enabled>
+                <inMemoryCached>true</inMemoryCached>
+                <name>opengeo:countries</name>
+                <mimeFormats>
+                  <string>image/png</string>
+                  <string>image/jpeg</string>
+                </mimeFormats>
+                <gridSubsets>
+                  <gridSubset>
+                    <gridSetName>EPSG:900913</gridSetName>
+                  </gridSubset>
+                  <gridSubset>
+                    <gridSetName>EPSG:4326</gridSetName>
+                  </gridSubset>
+                </gridSubsets>
+                <metaWidthHeight>
+                  <int>4</int>
+                  <int>4</int>
+                </metaWidthHeight>
+                <expireCache>0</expireCache>
+                <expireClients>0</expireClients>
+                <parameterFilters>
+                  <styleParameterFilter>
+                    <key>STYLES</key>
+                    <defaultValue/>
+                  </styleParameterFilter>
+                </parameterFilters>
+                <gutter>0</gutter>
+              </GeoServerLayer>
+        404:
+          description: Rest Exception 404 Unknown layer
+
+    put:
+      operationId: layersNamePut
+      summary: Create a new cached layer.
+      description: Creates a new cached layer on the server. Use the "Accept:" header to specify format or append an extension to the endpoint (example "/layers/{layer}.xml" for XML).
+      parameters:
+        - name: layerName
+          in: path
+          required: true
+          description: The name of the layer to add.
+          type: string
+        - name: layerBody
+          in: body
+          description: The new layer definition.
+          required: true
+          schema:
+            $ref: "#/definitions/Layer"
+      consumes:
+        - application/xml
+      responses:
+        201:
+          description: The layer was successfully created.
+        400:
+          description: Layer with requested name already exists, use POST if you want to replace it.
+
+    post:
+      operationId: layersNamePost
+      summary: Modify a cached layer.
+      description: Modifies an existing cached layer on the server. Use the "Accept:" header to specify format or append an extension to the endpoint (example "/layers/{layer}.xml" for XML).
```

Comment:
Since this is just text, probably want to add the base path too it, so no one gets confused with the geoserver layers endpoint. e.g. `/gwc/rest/layers/{layer}.xml`.

While that does make this a bit less applicable for standalone geowebcache, this is in the geoserver docs, so should probably be favouring embedded.